micro/service.go

```diff
@@ -258,7 +271,9 @@ func AddService(nc *nats.Conn, config Config) (Service, error) {
 		return nil, err
 	}
 
-	ping := Ping(svcIdentity)
+	ping := Ping{ServiceIdentity: svcIdentity,
+		Type: PingResponseType,
+	}
```

Comment:
This format of struct initialization is a bit hard to read IMO. It would be nice to have it as:
```go
	ping := Ping{
		ServiceIdentity: svcIdentity,
		Type: PingResponseType,
	}
```

Reply:
agree, gofmt missed this one will fix